integrationtests/cli/cleanup/suite_test.go

```diff
@@ -49,7 +49,7 @@ func (g *getter) GetNamespace() string {
 
 func TestFleetCLICleanUp(t *testing.T) {
 	RegisterFailHandler(Fail)
-	RunSpecs(t, "Fleet CLI Cleanup Suite")
+	RunSpecs(t, "GitJob Controller Suite")
```

Comment:
This seems wrong.
```suggestion
	RunSpecs(t, "Fleet CLI Cleanup")
```

Reply:
It was the other suite that needed correction.